eZ/Publish/Core/Persistence/Elasticsearch/Content/Search/Mapper/Standard.php

```diff
@@ -21,12 +21,16 @@
 use eZ\Publish\SPI\Persistence\Content\Type\Handler as ContentTypeHandler;
 use eZ\Publish\SPI\Persistence\Content\ObjectState\Handler as ObjectStateHandler;
 use eZ\Publish\SPI\Persistence\Content\Section\Handler as SectionHandler;
+use eZ\Publish\Core\Persistence\Elasticsearch\Content\Search\MapperInterface;
+use eZ\Publish\Core\Persistence\Elasticsearch\Content\Search\FieldNameGenerator;
+use eZ\Publish\Core\Persistence\Elasticsearch\Content\Search\Document;
 
 /**
- * Mapper maps Content and Location objects to a Document object, representing a
- * document in Elasticsearch index storage.
+ * Standard Mapper implementation maps:
+ *  - Content with its fields and corresponding Locations
+ *  - Locations with Content data but without Content fields
  */
-class Mapper
```

Comment:
You have a point there, we did agree not to use very short names.
Changed to `StandardMapper` and rebased.